cockroachcloud/managing-cmek.md

```diff
@@ -16,6 +16,12 @@ See also:
 - [Customer-Managed Encryption Key (CMEK) frequently asked questions (FAQ)](cmek-faq.html)
 - [Encryption at Rest (Enterprise)](../{{site.versions["stable"]}}/security-reference/encryption.html#encryption-at-rest)
 
+## Prerequisites
+
+- You need a {{ site.data.products.db }} account with billing enabled. Egress Perimeter Controls are available only for {{ site.data.products.dedicated }}, not {{ site.data.products.serverless }} clusters, and {{ site.data.products.dedicated }} have a non-zero operating cost that must be billed.
```

Comment:
Where does this point come from? This is not needed.

Reply:
I just misseed updating the feature name. I think this is true for CMEK as well, right?